x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/MachineLearningFlyout/index.tsx

```diff
@@ -49,14 +49,14 @@ export class MachineLearningFlyout extends Component<Props, State> {
       }
       this.addSuccessToast({ transactionType });
     } catch (e) {
-      this.addErrorToast();
+      this.addErrorToast(e as Error);
     }
 
     this.setState({ isCreatingJob: false });
     this.props.onClose();
   };
 
-  public addErrorToast = () => {
+  public addErrorToast = (error: Error & { body?: { message?: string } }) => {
```

Comment:
Can you make this a explicit type and move it to `x-pack/plugins/apm/public/services/rest/ml.ts`?
```
type MLError = Error & { body?: { message?: string } }
```